packages/react-components/priority-overflow/src/overflowManager.ts

```diff
@@ -72,6 +72,11 @@ export function createOverflowManager(): OverflowManager {
     return options.overflowAxis === 'horizontal' ? el.offsetWidth : el.offsetHeight;
   };
 
+  const getDividerLength = () => {
+    const divider = document.querySelector('[data-overflow-divider]');
```

Comment:
I don't like this way of getting divider. Maybe it would be better to register divider similar as it was done with items